src/libs/copy_history.c

```diff
@@ -156,6 +157,48 @@ static void copy_button_clicked(GtkWidget *widget, gpointer user_data)
   }
 }
 
+static void compress_button_clicked(GtkWidget *widget, gpointer user_data)
```

Comment:
Compress history is more complex now because of the mask history, check the darkroom compress.

Reply:
done